driver/Dockerfile.hail

```diff
@@ -3,8 +3,8 @@ FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.2
 
 ARG HAIL_SHA
 
-ENV HAIL_QUERY_BACKEND service \
-    TZ=Australia
+ENV HAIL_QUERY_BACKEND service
+ENV TZ Australia
```

Comment:
That's not even a valid timezone though, right? I suspect this was just a copy and paste issue or maybe was needed before `DEBIAN_FRONTEND=noninteractive` was added. Can we just remove this environment variable completely?